python/hail/methods/qc.py

```diff
@@ -231,31 +231,33 @@ def concordance(left, right):
 
     **Using the table results**
 
-    Table 1: Concordance statistics per column key
+    Table 1: Concordance statistics by column
 
-        - **s** -- column key.
-        - **n_discordant** (:py:data:`.tint64`) -- Count of discordant calls (see below for
+    This table contains the column key field of `left`, and the following fields:
+
+        - `num_discordant` (:py:data:`.tint64`) -- Count of discordant calls (see below for
           full definition).
-        - **concordance** (:class:`.tarray` of :class:`.tarray` of :py:data:`.tint64`) --
+        - `concordance` (:class:`.tarray` of :class:`.tarray` of :py:data:`.tint64`) --
           Array of concordance per state on left and right, matching the structure of
           the global summary defined above.
 
-    Table 2: Concordance statistics per row key
+    Table 2: Concordance statistics by row
+
+    This table contains the row key fields of `left`, and the following fields:
 
-        - **v** -- row key.
-        - **n_discordant** (:py:data:`.tfloat64`) -- Count of discordant calls (see below for
+        - `num_discordant` (:py:data:`.tfloat64`) -- Count of discordant calls (see below for
           full definition).
-        - **concordance** (:class:`.tarray` of :class:`.tarray` of :py:data:`.tint64`) --
+        - `concordance` (:class:`.tarray` of :class:`.tarray` of :py:data:`.tint64`) --
           Array of concordance per state on left and right, matching the structure of the
           global summary defined above.
 
-    In these tables, the column **n_discordant** is provided as a convenience,
+    In these tables, the column **num_discordant** is provided as a convenience,
```

Comment:
I thought everything was going to be `n_...`?

Reply:
this PR was made before that consensus, I'll update it